.vuepress/theme/Home.vue

```diff
@@ -146,7 +146,7 @@
                         </a>
                     </div>
                 </div>
-                <div class="footer">MIT Licensed | Pterodactyl&reg; Copyright &copy; 2015 - 2022 Dane Everitt & contributors.</div>
+                <div class="footer">MIT Licensed | Pterodactyl&reg; Copyright &copy; Dane Everitt and contributors</div>
```

Comment:
This is also not exclusive copyright to Dane anymore

Reply:
Well, pterodactyl/panel (and maybe) pterodactyl/wings still has Dane, so maybe we should keep it...?

Reply:
There was a commit recently by Matthew where he touched the license / copyright line in the about project page.
I would infer his preference from that.
https://github.com/pterodactyl/documentation/commit/cc2770a8a7661a9a1611338a003520be1eec7db4